newIDE/app/src/Profile/PublicProfileDialog.js

```diff
@@ -66,6 +66,7 @@ const PublicProfileDialog = ({ userId, onClose }: Props) => {
 
   return (
     <Dialog
+      title={<Trans>User profile</Trans>}
```

Comment:
Should we display the user name?

Reply:
I tried but the component below is also displaying the username, and is used in the profile.
Couldn't find an easy way to remove it from both places easily.
So I decided to remove this title for this particular dialog, it doesn't add value